spec/models/event_spec.rb

```diff
@@ -1,3 +1,135 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+
+RSpec.describe Fee, type: :model do
```

Comment:
RSpec/FilePath: Spec path should end with `fee*_spec.rb`. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FilePath)